src/scalecube-cluster/api/Cluster.js

```diff
@@ -0,0 +1,43 @@
+// @flow
```

Comment:
Maybe it would be better to put cluster api's in root api folder to centralize all the project interfaces in one place?

Reply:
@oarashi I want to delete the api in the root folder
I want each package will have it's own api.

Reply:
@idanilt 👍